home/ivp-planning/user-interface/the-2d-panel.md

```diff
@@ -1,33 +1,29 @@
 # 2D-Panel
 
-The 2D-Panel acts as planning area where you can [place](../machines/first-steps-with-3d-object.md), [move](../machines/selecting-and-moving-objects.md), [scale and rotate](../machines/scale-and-rotate-objects.md) machines and objects as if you would work with a traditional floor plan on paper.
+The 2D-Panel acts as planning area where you can [place](../machines/first-steps-with-3d-object.md), [move](../machines/selecting-and-moving-objects.md), [scale](../machines/scale-objects.md) and [rotate](../machines/rotate-objects.md) machines and objects as if you would work with a traditional floor plan on paper.
```

Comment:
rotate-objects.md does not exist in this branch. Should this change have been in #28?